test/unit/alphabet/composite/alphabet_variant_test.cpp

```diff
@@ -250,19 +250,19 @@ TEST(alphabet_variant_test, rank_type)
     EXPECT_TRUE((std::is_same_v<alphabet3_t::rank_type, uint16_t>));
 }
 
-TEST(alphabet_variant_test, value_size)
+TEST(alphabet_variant_test, alphabet_size)
```

Comment:
```suggestion
TEST(alphabet_variant_test, alphabet_size_)
```
?

Reply:
?

Reply:
I changed although I don't think it was necessary.